.github/containerscan/allowedlist.yaml

```diff
@@ -0,0 +1,7 @@
+general:
+  bestPracticeViolations:
+    # We violate this rule because we add kubectl from a remote location
+    # Instead of building it from source/copying it.
+    # Until we change our practices (e.g. have Dockerfile build kubectl
+    # in a multi-staged manner), we should skip this check
+    - CIS-DI-0009
```

Comment:
https://github.com/weaveworks/kured/issues/124 should also fix this AFAICS.

Reply:
correct, I was tackling that in parallel, but didn't get much time to work on it yet.